sources/api/netdog/src/networkd/devices/interface.rs

```diff
@@ -16,3 +19,70 @@ pub(crate) struct NetworkDInterface {
     pub(crate) static6: Option<StaticConfigV1>,
     pub(crate) routes: Option<Vec<RouteV1>>,
 }
+
+impl NetworkDInterface {
+    /// Does this interface contain any config?
```

Comment:
nit: this reads as a question from the author, like "What does this code do??"

`is_unconfigured()` might be better as a function name, and then the comment wouldn't be needed.